### ISSUE TITLE
403 when trying to "Sign into website" from Manyverse Android

### ISSUE THREAD

**Issue by luandro** [ssb-ngi-pointer/go-ssb-room#233]
### Device:
- Android 6
- Manyverse 0.2105.28-beta-indie
- Browser: Firefox Android 88.1.4

### Reproduce

- Join room (tried hermie.club and picoroom)
- Click on the room and click on ""Sign into website"
- Firefox opens up with the error:
 ![room_error](https://user-images.githubusercontent.com/693728/120214487-ed33f980-c20a-11eb-9d87-0dac5dab5a51.jpg)

**Comment by Powersource** [comment]
I get something very similar but instead the message

"Error #403 - Forbidden

The request could not be executed because of lacking privileges (ssb http auth: client not connected to room)"

**Comment by staltz** [comment]
To clarify, this issue is about *client-initiated* SSB HTTP Auth.

**Comment by Powersource** [comment]
> client-initiated SSB HTTP Auth.

Yes correct, I did the same flow as Luandro, clicked "Sign into website" in mv.

> Manyverse is killed by Android between the moment the app exits and the browser opens

That's likely to be the case for me. I "only" have [3GB of ram](https://www.gsmarena.com/sony_xperia_x_compact-8292.php) so it' very common I can't keep 2 apps going at the same time.

**Comment by staltz** [comment]
> > client-initiated SSB HTTP Auth.
> 
> Yes correct, I did the same flow as Luandro, clicked "Sign into website" in mv.
> 
> > Manyverse is killed by Android between the moment the app exits and the browser opens
> 
> That's likely to be the case for me. I "only" have [3GB of ram](https://www.gsmarena.com/sony_xperia_x_compact-8292.php) so it's very common I can't keep 2 apps going at the same time.

@Powersource Can you confirm it? Just try to go back to Manyverse as soon as you see the 403 error on the website. Let us know whether it shows the splash screen or if it shows the Connections tab.

**Comment by Powersource** [comment]
Yep splash screen. This worked pretty consistently last week so idk if something on my phone has changed (except for new mv version).

On second try now it worked. And logging in starting from the website worked as well now, I think that was pretty inconsistent before.

**Comment by Count042** [comment]
I'm having the same issue with Android 11.   When I switch back to manyverse, it remains on the connections tab, and does not launch again. When I try to initiate the login from firefox, it opens up manyverse and allows me to authenticate the login, but remains on the manyverse app.  If I try switch back to firefox, it continues to show "Waiting for Confirmation" And nothing happens.
